src/devtools/client/shared/components/splitter/SplitBox.css

```diff
@@ -91,3 +91,10 @@
 .split-box.dragging > .uncontrolled {
   pointer-events: none;
 }
+
+/* This disables splitter bar dragging when the user is 
+in the console. Dragging/resizing should only work in the debugger
+or the inspector where the split console can be displayed.  */
+#toolbox.console .split-box.horz > .splitter {
+  pointer-events: none;
+}
```

Comment:
I wonder if we need to add it to all 3 places if the CSS is *just* going to be redundant.

I think this is fine to do here, but we should probably remove the redundant CSS eventually...